src/app/cli/src/app.rs

```diff
@@ -436,10 +436,13 @@ fn configure_logging(output_config: &OutputConfig, workspace_layout: &WorkspaceL
         eprintln!(
             "\n{}",
             console::style(
-                "Oh no, looks like kamu has crashed!\n\
-                Please help us by reporting this problem at \
-                https://github.com/kamu-data/kamu-cli/issues"
-            ).bold()
+                "Oh no, looks like kamu has crashed!\n
+                Please try running `kamu system \
+                 diagnose` to verify your environment. \n
+                If the issue persists, help \
+                 us by reporting this problem at https://github.com/kamu-data/kamu-cli/issues"
```

Comment:
Should we mention `kamu system info` as well as a great source of information for issue reporting?